src/ompl/base/SpaceInformation.h

```diff
@@ -180,6 +180,11 @@ namespace ompl
                 return motionValidator_;
             }
 
+            MotionValidatorPtr& getMotionValidatorNonConst()
```

Comment:
I learned the ``NonConst`` standard from [moveit](https://github.com/ros-planning/moveit/blob/d001746257005e7bde01f1bed581ab78a763d3f0/moveit_core/planning_scene/include/moveit/planning_scene/planning_scene.h#L156) but am happy to change it